client/ayon_kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -12,7 +12,14 @@ class IntegrateKitsuNote(KitsuPublishContextPlugin):
 
     order = pyblish.api.IntegratorOrder
     label = "Kitsu Note and Status"
-    families = ["kitsu"]
+    families = ["kitsu", "render", "render.farm", "render.frames_farm",
+                "prerender", "prerender.farm", "prerender.frames_farm",
+                "renderlayer", "imagesequence", "image",
+                "vrayscene", "maxrender",
+                "arnold_rop", "mantra_rop",
+                "karma_rop", "vray_rop",
+                "redshift_rop", "usdrender"]
```

Comment:
if we have only `kitsu` so the script will not be triggered if we dont have peview but we need to, farm families wont work for me maybe i've done something wrong

Reply:
If it's a "hotfix" for you - then maybe you're for your specific use case better of creating a separate plug-in in e.g. your custom fork/branch outside of this PR that force adds the family `kitsu` to it.

To really fix that issue, the problem that the kitsu family isn't set - that is what we should solve instead. So that raises the question, why is `kitsu` family not set? OR the question becomes, why is the `kitsu` family required OR why can't it just run over ALL instances (`families = ["*"]`)?

Reply:
`(families = ["*"])` maybe this is ok idk i've just copied it from i tkink deadline plugin